Fedya/FANUC Client/RobotComunicationDifinition.inl

```diff
@@ -121,7 +97,7 @@ void RobotConnect<T>::sendParalel(std::mutex* mt, bool* f, RobotConnect* ins)
 				{
 					const RobotCoord rc = ins->_sendingQueue->front();
 					ins->_sendingQueue->pop();
-					ins->_robotSend->sendRobotCoord(rc);//���������� �����
+					ins->_robotSend->moveCoord(rc);//���������� �����
 				}
 				Sleep(1000);
```

Comment:
**(must be change, repeatedly)**: add `std::this_thread_sleep_for()` instead of this.